### PR TITLE
add TO006 multigas

### PR DESCRIPTION
**Pull request by rumachan**
Numerous changes required to add multigas sensors at TO006, close one temperature sensor, and change another temperature sensor.

### CODE REVIEW

**Comment by ricardojohansson** [comment]
Is this just waiting for a review @rumachan ?

**Comment by rumachan** [comment]
@ricardojohansson yes, have spoken to Elisabetta and she'll do it next week.

**Comment by elidana** [comment]
that's a big piece of work ! 

Some review notes here. Lots of changes to review, so I will update this comment as I go

- TO006 sensor code 03 closed - ok, and timestamp verified against data in the archive
-

File: environment/features.csv
Comment (elidana) [inline environment/features.csv:1058]: Hi Steve, should line n. 1058 be 
```diff
--TO006,08,01,H2S-conc,air,,2024-06-04T03:59:59Z,9999-01-01T00:00:00Z
++TO006,07,01,H2S-conc,air,,2024-06-04T03:59:59Z,9999-01-01T00:00:00Z
```
(sensor code for H2S is 07 and not 08??)

**Review by elidana — COMMENTED** [review]
will continue to review this later today, adding an initial comment

**Review by elidana — CHANGES_REQUESTED** [review]
Hi @rumachan ,

I finally completed this review.
A small typo, plus one thing to double check, and should be good to go :

- features file has a typo (see detailed comment)
- the internal ticket refer to a datalogger change, not sure if that happened, or if it was just a firmware upgrade (that is covered here) @ricardojohansson can you please confirm? recent file headers are showing this info (i.e. no datalogger change)
`"TOA5","4110","CR6","4110","CR6.Std.12.02","CPU:TO006-temaarifumarolesix-v007.CR6","22519","Data_Table"`

**Comment by rumachan** [comment]
Thanks for picking up 08 should have been 07 as channel number.

There was no instrument change. Richard confirmed to me before I did the work, and the TOA5 file confirms that.

Can you please re-review @elidana

**Review by elidana — APPROVED** [review]
looks good, thanks Steve! If you can please squash all commits before merging (can use the "squash and merge" button for that) that'd be great!

**Comment by rumachan** [comment]
Sorry, squash and merge is not enabled for this repository, it won't let me. So merging as is.

**Pull request merged**
